vars/dockerExecute.groovy

```diff
@@ -6,10 +6,10 @@ def call(Map parameters = [:], body) {
     def PLUGIN_ID_DOCKER_WORKFLOW = 'docker-workflow'
 
     handlePipelineStepErrors(stepName: STEP_NAME, stepParameters: parameters){
-        def dockerImage = parameters.get('dockerImage', '')
-        Map dockerEnvVars = parameters.get('dockerEnvVars', [:])
-        def dockerOptions = parameters.get('dockerOptions', '')
-        Map dockerVolumeBind = parameters.get('dockerVolumeBind', [:])
+        def dockerImage = parameters.dockerImage ?: ''
+        Map dockerEnvVars = parameters.dockerEnvVars ?: [:]
+        def dockerOptions = parameters.dockerOptions ?: ''
+        Map dockerVolumeBind = parameters.dockerVolumeBind ?: [:]
```

Comment:
what is the advantage of this change?

Reply:
get is not side effect free. It changes parameters.
http://docs.groovy-lang.org/latest/html/groovy-jdk/java/util/Map.html#get(java.lang.Object, java.lang.Object)

Reply:
Interesting, didn't know that!